tests/scheduler/node_test.go

```diff
@@ -82,7 +89,7 @@ func TestManualBackend(t *testing.T) {
 	go n.Run(ctx)
 
 	tasks := []string{}
-	for i := 1; i <= 10; i++ {
+	for i := 0; i < 1; i++ {
```

Comment:
I think more than one task should be run here

Reply:
Agreed, this was for debugging.